ocs/pod.py

```diff
@@ -17,30 +17,29 @@ class Pod(object):
     """Handles per pod related context
 
         Attributes:
-            name (str):      name of the pod in oc cluster
-            namespace(str):  openshift namespace where this pod lives
-            labels (list):   list of oc labels associated with pod
+            _name (str):      name of the pod in oc cluster
+            _namespace(str):  openshift namespace where this pod lives
+            labels (dict):   A dict of oc labels associated with pod
             roles (list):    This could be oc roles like Master, etcd OR
                              ceph roles like mon, osd etc
 
 
     """
 
-    def __init__(self, name=None, namespace=None, labels=None, roles=None):
+    def __init__(self, name=None, namespace=None, labels=None, roles=[]):
         """Context detail per pod
 
             Args:
                 name (string):      name of the pod in oc cluster
                 namespace (string): namespace in which pod lives
-                labels (list):      list of oc labels associated with pod
+                labels (dict):      list of oc labels associated with pod
```

Comment:
Dictionary of oc labels associated with pod

Reply:
Done